roles/confluent.common/tasks/secrets_protection.yml

```diff
@@ -22,6 +28,10 @@
     owner: "{{secrets_file_owner}}"
     group: "{{secrets_file_group}}"
```

Comment:
@erikgb This is where I came up with the whole backup/restore idea lol! I should test if this works with a `var`